tests/test_endpoints.py

```diff
@@ -16,64 +18,66 @@ def test_cli(loop, app, sanic_client):
     return loop.run_until_complete(sanic_client(app))
 
 
-async def test_index(test_cli):
-    resp = await test_cli.get('/')
-    assert resp.status == 200
+@pytest.fixture
+def generateData(request):
+    # adding 5 Language documents
+    insertDummyLanguages()
+
+    def fin():
+        removeCollections()
+
+    request.addfinalizer(fin)
 
 
-async def test_getLanguages_with_no_query_param(test_cli):
+async def test_getLanguages_with_noParam_shouldPass(test_cli, generateData):
     resp = await test_cli.get('/languages')
     resp_json = await resp.json()
-    assert resp_json["total_results"] == 0
+    assert resp_json["total_results"] == 5
+    assert len(resp_json["results"]) == 5
     assert resp_json["page"] == 1
     assert resp.status == 200
 
 
-async def test_getLanguages_with_more_query_param(test_cli):
-    params = {'page': 2, 'type': 'local', 'limit': 50}
+async def test_getLanguages_with_moreThanExpectedParam_shouldShowErrorMsg(test_cli, generateData):
```

Comment:
"show show error message" is not quite correct since you are not really showing anything. This an API. should fail is enough